docs/mainnet.md

```diff
@@ -0,0 +1,174 @@
+```bash
+./bin/swapcli take --peer-id 12D3KooWC547RfLcveQi1vBxACjnT6Uv15V11ortDTuxRWuhubGv \
+  --offer-id cf4bf01a0775a0d13fa41b14516e4b89034300707a1754e0d99b65f6cb6fffb9 --provides-amount 0.05
+# Initiated swap with ID=0
+```
+
+5. b. Alternatively, you can take the offer without getting notified of swap status updates:
+```bash
+./bin/swapcli take --peer-id 12D3KooWHLUrLnJtUbaGzTSi6azZavKhNgUZTtSiUZ9Uy12v1eZ7 \
+  --offer-id 0xcf4bf01a0775a0d13fa41b14516e4b89034300707a1754e0d99b65f6cb6fffb9 \
+  --provides-amount 0.05 --detached --swapd-port 5001
+```
+
+Either way, you can always check the swap's status via `./bin/swapcli ongoing`. 
+
+If all goes well, you should see the node execute the swap protocol. If the swap ends successfully, the XMR received will automatically be transferred back your original wallet.
+
+## Monero Maker
+
+1. Check your Monero address and balance and ensure your address is funded:
+```bash
+./bin/swapcli balances
+```
+
+2. a. Make an offer with `swapcli`:
+```bash
+./bin/swapcli make --min-amount MIN-XMR-AMOUNT --max-amount MAX-XMR-AMOUNT --exchange-rate EXCHANGE-RATE
+# Published offer with ID cf4bf01a0775a0d13fa41b14516e4b89034300707a1754e0d99b65f6cb6fffb9
+```
+
+> Note: the exchange rate is the ratio of XMR:ETH price. So for example, a ratio of 0.05 would mean 20 XMR to 1 ETH. You can see a suggested exchange rate from the Chainlink oracle using `swapcli suggested-exchange-rate`; however, you should always double check this against your own sources.
+
+3. b. Alternatively, make an offer with `swapcli` without subscribing to updates:
+```bash
+./bin/swapcli make --min-amount 0.1 --max-amount 1 --exchange-rate 0.5 --detached
+# Published offer with ID cf4bf01a0775a0d13fa41b14516e4b89034300707a1754e0d99b65f6cb6fffb9
+```
+
+When a peer takes your offer, you will see logs in `swapd` notifying you that a swap has been initiated. If all goes well, you'll receive the ETH in the account used by `swapd` and success logs in `swapd`. You can always check the swap's status via `swapcli ongoing` or `swapcli past`.
+
+## Troubleshooting
+
+Ideally, the exit case of the swap should be `Success`. If this is not the case, it will either be one of `Refunded` or `Aborted`.
+- `Refunded` means that the swap refunded after your funds were already locked. In this case, you would lose transaction fees only.
+- `Aborted` means that the swap exited before any funds were locked, so nothing was lost except time.
+
+Neither of these should happen, so if they happen, it indicates an issue either on your side or the remote peer's side.
+
+A few common errors are:
+- `Failed to get height`: double check that your `monerod` process is running.
+- `unlocked balance is less than maximum offer amount`: you will see this if you're a maker and try to make an offer but don't have enough balance. Either fund your account with more XMR or wait for your balance to unlock.
```

Comment:
Unfortunately, this can happen when you have enough XMR, but you made a recent monero transaction, and the majority of your balance is tied up in a change output waiting for 10 confirmations.  :|  That may be too specific to mention here though.

Reply:
I added a line mentioning that!